packages/api/src/core/index.ts

```diff
@@ -7,19 +7,29 @@ import fetchHar from 'fetch-har';
 import { FormDataEncoder } from 'form-data-encoder';
 import 'isomorphic-fetch';
 
+import FetchError from './errors/fetchError';
 import getJSONSchemaDefaults from './getJSONSchemaDefaults';
 import parseResponse from './parseResponse';
 import prepareAuth from './prepareAuth';
 import prepareParams from './prepareParams';
 import prepareServer from './prepareServer';
 
-export interface ConfigOptions {
-  /**
-   * By default we parse the response based on the `Content-Type` header of the request. You can
-   * disable this functionality by negating this option.
-   */
-  parseResponse: boolean;
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ConfigOptions {}
```

Comment:
Keeping this around because I'm going to need it when I start work on adding support for custom timeouts in https://github.com/readmeio/api/issues/432.